File: VB/DXWebApplication1/Controllers/HomeController.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Web.Mvc

Namespace DXWebApplication1.Controllers
	Public Class HomeController
		Inherits Controller
		Public Function Index() As ActionResult
			Return View()
		End Function

		<HttpPost, ValidateAntiForgeryToken> _
		Public Function CallbackPanelPartial() As ActionResult
			System.Threading.Thread.Sleep(1000)
			Return PartialView()
		End Function
	End Class
End Namespace

File: CS/DXWebApplication1/Controllers/HomeController.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace DXWebApplication1.Controllers {
    public class HomeController : Controller {
        public ActionResult Index() {
            return View();
        }

        [HttpPost]
        [ValidateAntiForgeryToken]
        public ActionResult CallbackPanelPartial() {
            System.Threading.Thread.Sleep(1000);
            return PartialView();
        }
    }
}